### ISSUE TITLE
picking terminal window: switching rows is very very slow

### ISSUE THREAD

**Issue by teosarca** [metasfresh/metasfresh-webui-frontend#1615]
### Is this a bug or feature request?
Bug

#### Which are the steps to reproduce?
* open picking terminal window: https://w101.metasfresh.com:8443/window/540345
* click around and remark that it takes ~2seconds to after you click on a row and the row gets selected. The backend is not the bottleneck in this case.

![image](https://user-images.githubusercontent.com/1244701/36484061-b39919a0-1720-11e8-89db-73da04af8acc.png)

NOTE: it seems something about included rows slows it down. If you check a regular window like `Sales Order` there is pretty fast.

### What is the expected or desired behavior?
Shall be instant.

**Comment by siemiatj** [comment]
Next thing on my plate

**Comment by metas-rc** [comment]
Results of IT

Tested on webui


* I made an order with 4 product lines, each on a TU.
* I opened Picking terminal from order
* OK: I clicked quickly from one row to another and they got selected instantly.

* I opened the Picking terminal for all the lines that are currently in the db
* OK: I clicked quickly from one row to another and they againg got selected instantly.
* OK: TU lines also selected quickly
*OK: The action was also changed from green to grey according to the line I selected
